assets/js/guides.js

```diff
@@ -137,4 +137,31 @@ $(document).ready(function () {
     }
   });
 
+  //Validate email
+  let validateEmail = email => {
+    let re = /@[\w.-]+\.[a-zA-Z]{2,6}$/;
```

Comment:
Generally, using a regex to validate email addresses is not a good idea. See:

https://davidcel.is/posts/stop-validating-email-addresses-with-regex/
https://stackoverflow.com/a/201378/483528

The validation you get from `<input type="email">` should be good enough... So what is the issue you're trying to solve?

Reply:
For some reason, if we enter a wrong email like this `ghtx@gm` without the last part of an email, the form allows us to submit but redirects to the previous newsletter page. The one that had the purple background and there the error is thrown. I am trying to capture the error once the `subscribe` button is clicked.

Reply:
> For some reason, if we enter a wrong email like this ghtx@gm without the last part of an email, the form allows us to submit but redirects to the previous newsletter page.

It allows submit because the `<input type="email">` validation allows that form of an email address, where as MailChimp's email validation does not. So as a result, MailChimp's validation redirects you to a MailChimp-hosted page (something like https://gruntwork.us13.list-manage.com/subscribe/post?u=8ccaad0747ac13944cf318e0f&id=786bd365d1) to correct the email address. The design of that page is, IIRC, defined in MailChimp itself. So perhaps the proper fix is not to try to emulate MailChimp's email validation (which could change at any time in the future), but to update the design of that page?

Reply:
There isn't so much that can be done but I changed the background color and the grunty image of the Mailchimp page to match what we currently use. Screenshot below
<img width="705" alt="Screenshot 2019-12-03 at 9 18 37 PM" src="https://user-images.githubusercontent.com/26963758/70086449-b7608580-1612-11ea-9525-9d52c2927590.png">

Reply:
This definitely looks better than before but consider using this Grunty: https://gruntwork.io/assets/img/guides/layout/grunty-boxes.png as the one you've used has that box pattern behind him and I wonder if it's supposed to be part of a bigger picture so that it doesn't look cut off?

Reply:
Done
<img width="688" alt="Screenshot 2019-12-03 at 11 03 04 PM" src="https://user-images.githubusercontent.com/26963758/70093640-2644db00-1621-11ea-9b91-5d568965837c.png">

Reply:
OK, that's a good improvement... But one issue: that red error color on top of the blue is very painful on the eyes ([it's a known issue with the human eye](https://www.blog.theteamw.com/2009/11/09/100-things-you-should-know-about-people-9-chromostereopsis-is-hard-on-the-eyes/)). 

Perhaps instead of the light blue background, we could go with the darker background on the gruntwork home page (`#1d2541`)? Or something else that contrasts better with red.